src/shipit_signoff/shipit_signoff/swagger/swagger.yaml

```diff
@@ -0,0 +1,190 @@
+---
```

Comment:
Did you mean to add a second copy of the swagger specs? There's api.yml above, too.